packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -173,6 +170,16 @@ export interface ContainerDefinitionProps {
   logging?: LogDriver;
 }
 
+/**
+ * Properties of a container definition
+ */
+export interface ContainerDefinitionProps extends ContainerDefinitionOptions {
+  /**
+   * The task this container definition belongs to.
+   */
+  task: TaskDefinition;
```

Comment:
`taskDefinition` please. *Task* has a distinct meaning and it means something else.

Reply:
Done